docs/guides/build-notifications.rst

```diff
@@ -1,13 +1,46 @@
 Enabling Build Notifications
 ============================
 
+Using Email
+******************
```

Comment:
Use `------` for subheadings with the same same size of the text, and should be sentence case.

You can read more about this here https://docs.readthedocs.io/en/latest/docs.html.